tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -150,14 +180,20 @@ def side_effect(_, temp_dir, *args, **kwargs):
     )
 
 
+@pytest.mark.parametrize('source_fbc, target_fbc', [(False, False), (False, True), (True, True)])
```

Comment:
Same question when source == FBC and target != FBC

Reply:
Already covered by https://github.com/release-engineering/iib/blob/50c78e92eb983c9fd3db76276c49c6b7b50cdfad/tests/test_workers/test_tasks/test_build_merge_index_image.py#L608